src/pages/MainPage/MainPage.tsx

```diff
@@ -18,10 +19,24 @@ const MainPage = () => {
     };
   }, {});
 
+  const optionsForSort = {
+    releaseDate: 'Release date',
```

Comment:
Looks like some styling is not working here.
Not critical though, just FYI.
![image](https://user-images.githubusercontent.com/25229823/178163050-f8d7958d-876e-4059-b530-056f1dd55336.png)